pkg/fileutil/embed.go

```diff
@@ -37,6 +40,16 @@ func CopyEmbedAssets(fsys embed.FS, sourceDir string, targetDir string) error {
 				if err != nil {
 					return err
 				}
+				if sigFound {
+					// If the file already exists and has the same content, don't overwrite it.
+					if existingContent, err := os.ReadFile(localPath); err == nil && string(existingContent) == string(content) {
```

Comment:
This seems good, but looks like even more work done on every command. It's better than what we're doing though. 

I guess our overwrite (current behavior) can never escape mutagen because it's probably looking at inodes, and we end up with new inodes.

Reply:
Yes, it does some extra work, but we can afford it since `#ddev-generated` files aren't too big, so it shouldn't slow DDEV down.